src/table.ts

```diff
@@ -1397,6 +1397,14 @@ class Table extends common.ServiceObject {
     }
 
     const dup = streamEvents(duplexify());
+    let uri: string;
+
+    if (this.bigQuery.baseUrl) {
+      const baseUrl = this.bigQuery.baseUrl.replace(/\/+$/, ''); // Remove trailing slashes
```

Comment:
Should this be done in the BigQuery constructor to avoid double-trailing slashes anywhere else as well? I think we probably have code somewhere upstream before a request goes out that would catch that, but it wouldn't hurt to clean up what we can when we get it.